whitesource/component.py

```diff
@@ -24,8 +25,12 @@ def _get_ws_label_from_artifact(source: cm.ComponentSource) -> dso.labels.Source
 
 
 def get_scan_artifacts_from_components(
-    components: typing.Sequence[cm.Component],
+    components: typing.Generator[typing.Union[tuple, typing.Any], typing.Any, typing.Any],
```

Comment:
I think `typing.Union[tuple, typing.Any], None, None` is the correct type, but please check